docs/en/02_Developer_Guides/08_Performance/02_HTTP_Cache_Headers.md

```diff
@@ -59,8 +59,8 @@ Does not set `private` directive, use `privateCache()` if this is explicitly req
 Simple way to set cache control header to a cacheable state.
 Use this method over `publicCache()` if you are unsure about caching details.
 
-Removes `no-store` and `no-cache` directives; other directives will remain in place.
-Use alongside `setMaxAge()` to indicate caching.
+Removes the `no-store` directive; other directives will remain in place.
```

Comment:
This still isn't quire true; it doesn't remove the `no-store` directive unless a max-age is supplied.

Reply:
Amended